adr/query_pushdown.md

```diff
@@ -0,0 +1,42 @@
+Select queries of the form `{_id: <ObjectID>}` (value type is `ObjectID`).
+
+## Postgres
+
+If value type is not `ObjectID`, fallback to fetch entire table.
+If value type is `ObjectID` raise error (`fjson` unmarshal).
+
+Both work:
+
+```sql
+select * from test where (_jsonb->'_id')::jsonb->>'$o' = '507f1f77bcf86cd799439011';
+```
+
+```sql
+select * from test where _jsonb->'_id' = '{"$o":"507f1f77bcf86cd799439011"}'::jsonb;
+```
+
+[PostgreSQL functions](https://www.postgresql.org/docs/14/functions-json.html)
+
+## Tigris
+
+Support tables where the primary key is only one field.
+
+* `if len(schema.PrimaryKey) > 1` fallback to fetch the entire table.
+* vlaue type if not `ObjectID` raise error.
```

Comment:
What do you mean by `ObjectID`? A specific type, a set of types? I don't see `ObjectID` mentioned in the article linked above.